src/components/EventCard/FavoriteButton.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component, Fragment } from 'react';
+import { inject, observer } from 'mobx-react';
+import { withRouter } from 'react-router';
+import {
```

Comment:
nit: move to the same line. I disabled this rule for eslint so it doesn't apply the max-len rule for imports.